src-env (msdb)/_MicroserviceTemplate_.WebApi/Controllers/MyTableController.cs

```diff
@@ -0,0 +1,155 @@
+﻿using _MicroserviceTemplate_.Domain.MyTables;
+using _MicroserviceTemplate_.WebApi.Authorization;
+using _MicroserviceTemplate_.WebApi.Models.MyTables;
+using AutoMapper;
+using Benraz.Infrastructure.Web.Filters;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace _MicroserviceTemplate_.WebApi.Controllers
+{
+    /// <summary>
+    /// My table controller.
+    /// </summary>
+    [ApiController]
+    [Route("/v{version:ApiVersion}/[controller]")]
+    public class MyTableController : ControllerBase
+    {
+        private readonly IMyTablesRepository _myTablesRepository;
+        private readonly IMapper _mapper;
+
+        /// <summary>
+        /// Creates controller.
+        /// </summary>
+        /// <param name="myTablesRepository">My tables repository.</param>
+        /// <param name="mapper">Mapper.</param>
+        public MyTableController(IMyTablesRepository myTablesRepository, IMapper mapper)
+        {
+            _myTablesRepository = myTablesRepository;
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// Returns my table entry.
+        /// </summary>
+        /// <param name="id">My table identifier.</param>
+        /// <returns>my table entry.</returns>
+        [HttpGet("{id}")]
+        [Authorize(_MicroserviceTemplate_Policies.MYTABLES_READ)]
+        [ProducesResponseType(typeof(MyTableViewModel), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetByIdAsync([FromRoute] string id)
```

Comment:
ID IS GUID all our tables have id GUID, the only one was settings which the idea was to remove this setting table.